portfolio/src/main/webapp/contact.html

```diff
@@ -26,32 +26,33 @@ <h1 class="header">CONTACT ME</h1>
 
         <div id="commentArea">
             <h2>COMMENT BELOW</h2>
-            <div class="commentCol">
-                <div id="commentForm">
-                <form action="/new-comment" method="post">
-                        <p>NAME</p>
-                        <input type="text" id="name" name="name" required> 
-                        <p>COMMENT</p>
-                        <input name="text-input" required>
-                        <br/><br/>
-                        <input type="submit" value="SUBMIT" class="button"/>
-                        <br/><br/>
-                        <p>DISPLAY OPTIONS</p>
-                        <label for="limit">COMMENT LIMIT</label>
-                        <input type="number" id="limit" name="limit" value=5> 
-                        <label for="sort">SORT BY:</label>
-                        <select name="sort" id="sort" class="button smallDefaultButton">
-                            <option value="descending">NEWEST</option>
-                            <option value="ascending">OLDEST</option>
-                        </select>
-                        <br/><br/>
-                        <label for="limit">FILTER NAME</label>
-                        <input type="text" id="searchName" name="searchName">
-                        <button onclick="getComment();return false;" class="smallDefaultButton">REFRESH</button>
-                    </form>
+            <div id="login-container"></div>
+            <form>
```

Comment:
I don't think you actually need the `form` element here, unless you're using it for styling.